hm9_1.py

```diff
@@ -0,0 +1,30 @@
+""" Task 1: clean # """
+
+
+def line_with_symbol(line):
+
+    """ Function cleans symbols '#' and the symbol before """
+
+    new_line = line[:]
```

Comment:
You do not need to cope a list, work with `line`

Reply:
See you, fixed 👍